### PR TITLE
feat: add сallMe MVC component

### PR DESCRIPTION
**Pull request by MarinaPasternak**

### CODE REVIEW

File: app/components/callMe/call-styles.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import '../../css/variables';
+
+
+.pouop-conteiner .form-show{
```
Comment (rearchontis) [inline app/components/callMe/call-styles.scss]: Strange selector. Should it be `.popup-container`?

File: app/components/callMe/call-styles.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    margin: auto;
+    border-radius: 10px;
+    display: grid;
+    height: fit-content;
+    padding: 20px;
+    background-color: rgb(209, 202, 230);
+} 
+
+.pouop-conteiner input[type="text"]{
```
Comment (rearchontis) [inline app/components/callMe/call-styles.scss]: Rewrite selector, please.

File: app/components/callMe/call-styles.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    bottom: 20%;
+    cursor: pointer;
+    z-index: 2;
+    overflow: hidden;
+    display: flex;
+    flex-direction: row;
+    align-items:flex-end;
+}
+
+.pouop-conteiner p
```
Comment (rearchontis) [inline app/components/callMe/call-styles.scss]: The curly bracket at a new line, but others aren't.

File: app/components/callMe/call-styles.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    border-radius: 25px;
+    margin: 10px;
+}
+
+
+.btn:hover {
+    background-color:  rgb(156, 141, 202);
+}
+
+.pouop-conteiner{
```
Comment (rearchontis) [inline app/components/callMe/call-styles.scss]: Rewrite selector.

File: app/components/callMe/call-styles.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+.call-me-btn{
+    background-color:  rgb(126, 139, 230);
+    box-shadow: 3px 3px 3px rgba(0,0,0,0.2);
+    margin: auto;
+    padding: 11px;
+    border: 0;
+
+}
+
+.btn {
```
Comment (rearchontis) [inline app/components/callMe/call-styles.scss]: Space between the selector and the curly bracket, but usually they absent. Why?

**Comment by Anastasiia0999** [comment]
- **Pls change to data.replace(/[^\d]/g, '');**
- **put ; after arrow func**

image

**Review by Anastasiia0999 — COMMENTED** [review]
**what about connection with bot?**
image

**Comment by rearchontis** [comment]
> **what about connection with bot?**
> image

It's my job :D
But I'll be able to apply bot just after merging Call Me into dev.

File: app/components/callMe/call-model.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+export class CallModel {
+    validation = (data) => {
+      const callNumber = data.target.value.replace(/[^\d]/g, '');
```
Comment (PavelKundenko) [inline app/components/callMe/call-model.js]: You can simplify this function: `validation = (data) => data.replace(/[^\d]/g, '');`

**Review by Anastasiia0999 — CHANGES_REQUESTED** [review]
**Why do u put this.model.validation and use handleFormSubmit in view? Change it to this.handleFormSubmit**
image
image

**Review by Anastasiia0999 — CHANGES_REQUESTED** [review]
Pls, change it => data.replace
image

**Review by Anastasiia0999 — APPROVED** [review]

**Review by PavelKundenko — APPROVED** [review]

**Review by rearchontis — APPROVED** [review]

**Pull request merged**